includes/definitions/discovery/netagent2.yaml

```diff
@@ -0,0 +1,60 @@
+mib: UPS-MIB:XPPC-MIB
+modules:
+    sensors:
+        state:
+            data:
+                -
+                    oid: upsBaseBatteryStatus
+                    num_oid: .1.3.6.1.4.1.935.1.1.1.2.1.1.
+                    descr: Battery Status
+                    index: 'upsBaseBatteryStatus.{{ $index }}'
+                    state_name: upsBaseBatteryStatus
+                    states:
+                        - { descr: unknown, graph: 0, value: 1, generic: 3 }
+                        - { descr: BatteryNormal, graph: 0, value: 2, generic: 0 }
+                        - { descr: BatteryLow, graph: 0, value: 3, generic: 1 }
+                -
+                    oid: upsSmartBatteryReplaceIndicator
+                    num_oid: .1.3.6.1.4.1.935.1.1.1.2.2.5.
+                    descr: Battery Replace Status
+                    index: 'upsSmartBatteryReplaceIndicator.{{ $index }}'
+                    state_name: upsSmartBatteryReplaceIndicator
+                    states:
+                        - { descr: noBatteryNeedsReplacing, graph: 0, value: 1, generic: 0 }
+                        - { descr: batteryNeedsReplacing, graph: 0, value: 2, generic: 2 }
+                -
+                    oid: upsBaseOutputStatus
```

Comment:
This was the existing sensor moved from php, because the index and state name has changed then people will lose data, you should keep them the same here for backwards compatibility (unless the previous way was broke).